frontend-ssr/redux/exercise-submissions/exercise-submissions.reducer.ts

```diff
@@ -0,0 +1,72 @@
+import { SubmissionState } from './types';
+import { EXERCISE_SUBMISSIONS } from './exercise-submissions.actions';
+
+const initialState: SubmissionState = {
+  submissions: undefined,
+  search: '',
+  page: 1,
```

Comment:
I see you changed the page number at the botttom, but you should have changed it here since 0 is the starting one.